libsolidity/interface/CompilerStack.cpp

```diff
@@ -769,6 +770,13 @@ h256 const& CompilerStack::Source::swarmHash() const
 	return swarmHashCached;
 }
 
+string const& CompilerStack::Source::ipfsUrl() const
+{
+	if (ipfsUrlCached.empty())
+		if (scanner->source().size() < 1025 * 256)
+			ipfsUrlCached = "dweb:/ipfs/" + dev::ipfsHashBase58(scanner->source());
```

Comment:
I believe this is the recommended way to denote IPFS URIs as per https://github.com/ipfs/specs/pull/152#issuecomment-284628862

Reply:
It depends on which "upgrade stage" we want to be ;)

Reply:
We cannot use `/ipfs` because we need a scheme, and I think we can afford not using `ipfs://` because this is just a json artefact and not a website.

Reply:
Ok